webpack/containers/Header.js

```diff
@@ -9,9 +9,18 @@ import NotificationDropdown from '../components/NotificationDropdown';
 import NotificationMenu from '../components/NotificationMenu';
 import { fetchNotifications } from '../actions/notification_actions';
 
-let LoginMenu = ({logInOpener, signUpOpener, currentUser}) => {
+let LoginMenu = ({openIdServer, logInOpener, signUpOpener, currentUser}) => {
   let loggedIn = ! _.isEmpty(currentUser);
   if(!loggedIn) {
+    if(openIdServer != '') {
+      return (
+        <ul className="nav navbar-nav">
+          <li className="log-in-link">
+            <a tabIndex="2" href={openIdServer}>Login</a>
```

Comment:
This was based on a few assumptions about what the env vars would point to. I was trying to reuse as much as possible to avoid other config steps. However, if the OPIDC_HOST is not the URL for the login I can make the env variable that instead.